pkgs/applications/editors/emacs/elisp-packages/manual-packages/acm-terminal/default.nix

```diff
@@ -1,55 +1,34 @@
-{ lib
-, melpaBuild
-, fetchFromGitHub
-, acm
-, popon
-, writeText
-, writeScript
+{
+  lib,
+  acm,
+  fetchFromGitHub,
+  melpaBuild,
+  popon,
+  unstableGitUpdater,
 }:
 
-let
-  rev = "1851d8fa2a27d3fd8deeeb29cd21c3002b8351ba";
-in
 melpaBuild {
```

Comment:
Well, yes, but I do not think this is needed.
In my opinion, the change is small enough to be understandable.

If you prefer, I can do this.

Reply:
Closing as solved.

Reply:
I was just thinking that if this PR was going to be blocked it would be easier to merge it in smaller chunks, but this looks good to me, so I'd say don't bother!